docs/installation/docker.md

```diff
@@ -2,10 +2,9 @@
 title: Docker
 ---
 
-# Installing DEV with Docker [Beta]
+# Installing DEV with Docker
```

Comment:
With all the work that went into this over the past few months, I don't want to call this "Beta" anymore.  Also slightly reworded the call to action re issues below.